src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -192,6 +197,25 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
   const focusId = buildFocusIdForUrl({ type: 'gene', objectId: geneId });
   const gbUrl = urlFor.browser({ genomeId, focus: focusId });
 
+  const isSidebarOpen = useSelector(isEntityViewerSidebarOpen);
+
+  useEffect(() => {
+    if (!genomeId || !props.gene) {
+      return;
+    }
+
+    if (isSidebarOpen) {
+      dispatch(closeSidebarModal());
+    }
+
+    dispatch(
+      updatePreviouslyViewedEntities({
+        genomeId,
+        gene: props.gene
+      })
+    );
```

Comment:
Typescript in the CI pipeline is complaining — I suppose the problem is here?

![image](https://user-images.githubusercontent.com/6834224/109577170-0e425c80-7aed-11eb-8d64-1eb734d83b4a.png)

Reply:
Yes thats right. Just did a fix :-)

Reply:
Sorry, the fixed changed the wrong type :-)

What the type error was saying was that the gene type, as we declared it it in the GeneView component, doesn't have a `symbol` field. But we are requesting the `symbol` field in the graphql query. Therefore, our definition of the Gene type is incorrect: we should update this: https://github.com/Ensembl/ensembl-client/blob/df8c1c79c3b92a50743bbcc6744ca2f7768aef52/src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx#L60-L62

to read:

```ts
import { FullGene } from 'src/shared/types/thoas/gene';

...


type Gene = Pick<FullGene, 'symbol'> &
  GeneOverviewImageProps['gene'] &
  DefaultTranscriptsListProps['gene'] &
  GeneFunctionProps['gene'];
```

while the type of `UpdatePreviouslyViewedPayload` should remain to what it was — we want to always pass the `symbol` field (which can be string or null) to the updatePreviouslyViewed function.

Reply:
Thanks, updated 👍